fairer-iOS/fairer-iOS/Global/Literal/ImageLiteral.swift

```diff
@@ -12,7 +12,10 @@ enum ImageLiterals {
     // MARK: - image
 
     static var imgLogo: UIImage { .load(name: "fairerlogo")}
+    static var imgLogoLogin: UIImage { .load(name: "fairerlogologin")}
     static var imgLogoSplash: UIImage { .load(name: "fairerlogosplash")}
+    static var appleLogo: UIImage { .load(name: "applelogo")}
+    static var googleLogo: UIImage { .load(name: "googlelogo")}
```

Comment:
p4;

위와 통일되게 img를 붙이는건 어떨까요 ??
```suggestion
    static var imgAppleLogo: UIImage { .load(name: "applelogo")}
    static var imgGoogleLogo: UIImage { .load(name: "googlelogo")}
```

Reply:
[48b3df0](https://github.com/fairer-iOS/fairer-iOS/pull/25/commits/48b3df0e9225a7dbda93a94eb8fc2aa8c1a130b7)
수정해서 반영해두었습니다 ! 감사합니당 !!!!!